src/components/fields/MultiColorPicker.js

```diff
@@ -8,6 +8,24 @@ import React, {Component} from 'react';
 import nestedProperty from 'plotly.js/src/lib/nested_property';
 import {adjustColorscale, connectToContainer} from 'lib';
 
+const CustomColorscalePicker = connectToContainer(UnconnectedColorscalePicker, {
+  modifyPlotProps: (props, context, plotProps) => {
+    if (
+      props.attr === 'marker.color' &&
+      context.fullData
+        .filter(t => context.traceIndexes.includes(t.index))
+        .every(t => t.marker && t.marker.color) &&
+      (plotProps.fullValue && typeof plotProps.fullValue === 'string')
+    ) {
+      plotProps.fullValue =
+        context.fullData &&
+        context.fullData
+          .filter(t => context.traceIndexes.includes(t.index))
+          .map(t => [0, t.marker.color]);
```

Comment:
so for the same reason as for pies below, I think we no longer want the `.map`

Reply:
Again this used to be “&& is marker.color” so I think it’s still needed

Reply:
same as above, my comment was just about the `.map` part :)